rest_framework_jwt/views.py

```diff
@@ -124,7 +154,7 @@ def get(self, request):
         return Response({}, status=status.HTTP_200_OK)
```

Comment:
PLeasae also do not leave it as "200 OK" - just raise exception always until it is implemented.